.github/workflows/release.yml

```diff
@@ -1,115 +1,30 @@
-name: Build and Release Process
-
-# Defines environmental variables
-env:
-  GO_VERSION: '1.22.2'
+name: goreleaser
```

Comment:
Maybe something more descriptive here since we have 'goreleaser' down below. "Release Festerize" or "Release Client" or just "Build and Release"?